### PR TITLE
Add 'mymuesli GmbH' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.netlify.com/#!doc=%7B%22slug%22%3A%22mymuesli%22%2C%22relevant-countries%22%3A%5B%22de%22%2C%22ch%22%2C%22nl%22%2C%22be%22%2C%22gb%22%2C%22se%22%2C%22pl%22%2C%22fr%22%2C%22at%22%5D%2C%22categories%22%3A%5B%22commerce%22%5D%2C%22name%22%3A%22mymuesli%20GmbH%22%2C%22address%22%3A%22Sailerw%C3%B6hr%2016%5Cn94032%20Passau%5CnGermany%22%2C%22phone%22%3A%22%2B49%20851%2020426699%22%2C%22fax%22%3A%22%2B49%20851%2020426693%22%2C%22email%22%3A%22dataprotection%40mymuesli.com%22%2C%22web%22%3A%22https%3A%2F%2Fwww.mymuesli.com%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.mymuesli.com%2Fdatenschutz%22%2C%22https%3A%2F%2Fwww.mymuesli.com%2Fimpressum%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F952%22%5D%2C%22required-elements%22%3A%5B%7B%22desc%22%3A%22Name%22%2C%22type%22%3A%22name%22%2C%22optional%22%3Afalse%7D%2C%7B%22desc%22%3A%22E-Mail%20address%22%2C%22type%22%3A%22email%22%2C%22optional%22%3Afalse%7D%2C%7B%22desc%22%3A%22Address%22%2C%22type%22%3A%22address%22%2C%22optional%22%3Atrue%7D%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

**Review by mal-tee — COMMENTED** [review]
Thanks!

I've made some major changes, @baltpeter would you take a look?

File: companies/mymuesli.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            "type": "email",
+            "optional": false
+        },
+        {
+            "desc": "Address",
+            "type": "address",
+            "optional": true
+        }
+    ],
+    "request-language": "de",
```
Comment (baltpeter) [inline companies/mymuesli.json:46]: From what I can see, the different country versions are translated, so I don't see a need to specify that.

File: companies/mymuesli.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "suggested-transport-medium": "email",
+    "comments": [
+        "You can reach the data protection officer at dsb.mymuesli@verimax.de",
+        "The phone number above is the data protection officer's. You can reach mymuesli directly at +49 851 20426699"
```
Comment (baltpeter) [inline companies/mymuesli.json:50]: There is a webform that we could link: https://www.mymuesli.com/contact/subject/14 (not sure about i18n, though…)

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**